src/main/java/sirius/db/mongo/Mango.java

```diff
@@ -160,7 +162,19 @@ private void writeField(MongoEntity entity, Updater updater, Property property)
      * @return <tt>true</tt> if the given value is a default value, <tt>false</tt> otherwise
      */
     private boolean isDefaultValue(Object valueForDatasource) {
-        return valueForDatasource == null || Boolean.FALSE.equals(valueForDatasource);
+        if (valueForDatasource == null) {
+            return true;
+        }
+
+        if (valueForDatasource instanceof List && ((List<?>) valueForDatasource).isEmpty()) {
```

Comment:
```suggestion
        if (valueForDatasource instanceof Collection && ((Collection<?>) valueForDatasource).isEmpty()) {
```
... that is, if Mongo supports collections other than lists. If this is not the case, it does not matter, obviously

Reply:
Actually is not about mongo, but the object underlying our properties.